src/KM_Game.pas

```diff
@@ -628,6 +631,18 @@ procedure TKMGame.GameMPReadyToPlay(Sender: TObject);
 end;
 
 
+procedure TKMGame.Game_OnReassignedHost(aDefeatedPlayerHandId: Integer);
+begin
+  OtherPlayerDisconnected(aDefeatedPlayerHandId);
```

Comment:
As discussed this is not correct. Game_OnReassignedHost can be removed. Is fNetworking.OnReassignedHost unused then? Or does lobby bind it? The event can be removed if it's unused.